MnemonicKit.xcodeproj/xcshareddata/xcschemes/MnemonicKitTests.xcscheme

```diff
@@ -10,7 +10,6 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      codeCoverageEnabled = "YES"
```

Comment:
I think this line is important because it generates the CodeCoverage file that is used by Slather and codecov.io

Reply:
Yeah, you're right, I have deleted and re-added the test target to fix some issue and have not noticed this had got deleted, so thanks for noticing 🎊